dev-util/drone/files/droned.init

```diff
@@ -0,0 +1,75 @@
+#! /bin/sh
```

Comment:
Hmm... this doesn't seem to be Gentooish so much. Can you rewrite using runscript?